token_contract/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "soroban-token-contract"
+description = "Soroban standard token contract"
+version = "0.0.4"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["export"]
+export = []
+testutils = ["soroban-sdk/testutils", "soroban-auth/testutils", "dep:ed25519-dalek"]
+
+[dependencies]
+ed25519-dalek = { version = "1.0.1", optional = true }
+num-bigint = { version = "0.4", optional = true }
+soroban-sdk = { version = "0.0.4" }
+soroban-auth = { version = "0.0.4" }
+
+[dev-dependencies]
+soroban-token-contract = { path = ".", features = ["export", "testutils"] }
```

Comment:
This self dependency needs removing. One of the builds is breaking showing that it is not compatible with rust-analyzer: 
```
[ERROR project_model::workspace] cyclic deps: soroban_token_contract(CrateId(98)) -> soroban_token_contract(CrateId(98)), alternative path: soroban_token_contract(CrateId(98))
```

Reply:
Removed